en/best-practices.md

```diff
@@ -236,17 +238,18 @@ Eventually, the GTFS Best Practices will become part of the core GTFS Reference.
 
 ### How to check for conformance with these Best Practices?
 
-The Canonical GTFS Schedule Validator checks for compliance against these Best Practices. You can find more about this validation tool on the [validate page](https://gtfs.org/schedule/validate/).
+The Canonical GTFS Schedule Validator checks for compliance against the GTFS Best Practices. Each WARNING corresponds to recommendations that are either explicitly suggested by the GTFS Schedule Reference, using the term “recommend” or “should,” or mentioned in this document.
```

Comment:
The validator checks for ALL Best Practices.  Some can only checked manually - those should be specified. 

Are there warnings that aren't best practices in the validator?

Reply:
If we want to be exact here:
- the validator checks for all Best Practices (from this doc and the spec) _that can be checked automatically_. Certain Best Practices can't, so I've updated the statement to represent this. Note that it's the same on the spec.
- certain validator warnings (1 or 2) are not associated with a spec or Best Practice statement, because we considered them important to be part of the tool, although difficult to represent in spec language because they are cases where the data is valid, but contains something very weird that looks unintentional. The validator documentation explains the provenance of each notice.

Reply:
I updated the paragraph to make it clearer that the validator checks for BP which can be checked automatically. 

Providing a list of BP (and spec statements) that aren't checked by the canonical validators as a "guide to GTFS compliance" would be great indeed, but it's out of scope for this particular PR.